src/oom/oomd.c

```diff
@@ -21,13 +21,50 @@
 static bool arg_dry_run = false;
 static int arg_swap_used_limit_permyriad = -1;
 static int arg_mem_pressure_limit_permyriad = -1;
-static usec_t arg_mem_pressure_usec = 0;
+static usec_t arg_mem_pressure_usec = DEFAULT_MEM_PRESSURE_DURATION_USEC;
+
+static int config_parse_duration(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        usec_t usec, *duration = ASSERT_PTR(data);
+        int r;
+
+        if (isempty(rvalue)) {
+                *duration = DEFAULT_MEM_PRESSURE_DURATION_USEC;
+                return 0;
+        }
+
+        r = parse_sec(rvalue, &usec);
+        if (r < 0)
+                return log_syntax_parse_error(unit, filename, line, r, lvalue, rvalue);
+
+        if (usec == 0) {
+                /* Map zero -> default for backwards compatibility. */
+                *duration = DEFAULT_MEM_PRESSURE_DURATION_USEC;
+                return 0;
+        }
+
+        if (usec < 1 * USEC_PER_SEC || usec == USEC_INFINITY)
+                return log_syntax(unit, LOG_ERR, filename, line, SYNTHETIC_ERRNO(EINVAL), "%s= must be at least 1s and less than infinity", lvalue);
```

Comment:
Returning error code cause the `config_parse_standard_file_with_dropins()` fails, and prohibit to oomd started.
Please gracefully handle the misconfiguration. So, `SYNTHETIC_ERRNO(EINVAL)` -> `0`.

Reply:
Ah I was under the impression `run()` in oomd.c will fail to start if misconfigured. But seems we are changing that here to handle misconfiguration gracefully (which makes sense, oomd not starting is probably worse and is nicer for backwards compatibility)

Reply:
BTW, insert a full stop at the end of the log message.

Reply:
Let me also change this to LOG_WARNING since we're ignoring the value if misconfigured. Will change it to similar to load-fragment.c:
```
 log_syntax(unit, LOG_WARNING, filename, line, 0, "%s= must be at least 1s and less than infinity, ignoring: %s", lvalue, rvalue);
```

Reply:
Yeah, LOG_WARNING is better.

Reply:
Also re: hard stop, all other logs I see ending with "ignoring: %s" (or something akin to that) don't have a period at the end. Likely because it'll be confusing for users thinking the value ends with a period and there is a parsing error. So not adding a period but lemme know if I'm wrong

Reply:
That's much nicer!